src/System.Private.Xml/tests/XmlSchema/XmlSchemaSet/TC_SchemaSet_Compile.cs

```diff
@@ -118,33 +118,42 @@ public void TFS_470021()
     </xs:complexType>
   </xs:element>
 </xs:schema>";
-            using (XmlWriter w = XmlWriter.Create("cham.xsd"))
+
+            using (var tempDirectory = new TempDirectory())
             {
-                using (XmlReader r = XmlReader.Create(new StringReader(cham)))
-                    w.WriteNode(r, true);
-            }
-            XmlSchemaSet ss = new XmlSchemaSet();
-            ss.ValidationEventHandler += new ValidationEventHandler(ValidationCallback);
+                string chamPath = Path.Combine(tempDirectory.Path, "cham.xsd");
+                string tempDirectoryPath = tempDirectory.Path[tempDirectory.Path.Length - 1] == Path.DirectorySeparatorChar ?
```

Comment:
I am confused now. With your latest commit, shouldn't the one test be passing now on e.g. Ubuntu? It still says all three tests failed